index.js

```diff
@@ -118,6 +120,14 @@ const parseArgs = ({
     }
   );
 
+  const checkSafeOptionValue = (optionAsUsed, longOptionName, value) => {
+    if (strict && !isSafeOptionValue(value)) {
+      const errorMessage = `Did you forget to specify the option argument for '${optionAsUsed}'?
+To specify an option argument starting with a dash use '--${longOptionName}=-EXAMPLE'.`;
+      throw new Error(errorMessage);
```

Comment:
**suggestion (non-blocking):** Perhaps inline the error message since it's not being referenced anywhere else.

Reply:
The line of code is visible in the resulting output when the error is thrown, so the variable was to make that tidier.